src/internal/model_render.go

```
@@ -253,71 +253,101 @@ func (m *model) processBarRender() string {
+func sortMetadata(meta [][2]string) [][2]string {
+	priority := map[string]int{
+		"Name":          0,
+		"Size":          1,
+		"Date Modified": 2,
+		"Date Accessed": 3,
+	}
+
+	sort.SliceStable(meta, func(i, j int) bool {
+		pi, iok := priority[meta[i][0]]
+		pj, jok := priority[meta[j][0]]
+		if iok && jok {
```

golangci-lint (1.64.8)

ifElseChain: rewrite if-else to switch statement

(gocritic)